index.html

```diff
@@ -317,7 +330,7 @@ <h2 id="project_tagline">eiffel-remrem-generate</h2>
 <pre>curl -H "Content-Type: application/json" -X POST -d '
   {"msgParams": {
   "meta": {
-    "type": "eiffelactivityfinished",
+    "type": "EiffelActivityFinishedEvent",
```

Comment:
mention in documentation about that type is not mandatory for eiffel-remrem-semantics it will be added based on event type query, but if present it should be Camelcase.